web/src/page_params.ts

```diff
@@ -1,7 +1,17 @@
 import $ from "jquery";
 
+type ContributorProfile = {
+    avatar: string;
+    email: string;
+    github_username?: string;
+    name: string;
+};
+type RepositoryDetails = {[key: string]: number};
+export type Contributor = ContributorProfile & RepositoryDetails;
+
 const t1 = performance.now();
 export const page_params: {
```

Comment:
We might not want to modify the `page_params` for the main application, the `page_params` for `teams.ts` is only available for [this view](https://github.com/zulip/zulip/blob/main/corporate/views/portico.py#L86-L106). Instead, all the related types should live in `teams.ts`.

Reply:
I think we can refactor away the usage of `page_params` from `team.js` ([some relevant info](https://chat.zulip.org/#narrow/stream/6-frontend/topic/page.20params/near/1455773)), by having the caller of `render_tabs` pass in the contributor's data directly. So that `page_params.ts` don't need to be changed or depend on `team.ts`.

The way this can be achieved is similar to [this commit](https://github.com/zulip/zulip/pull/23927/commits/63dcab27676092cd09e506b7303a8053f98a4a12).